framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/builder/WorkbenchKieServerPersistentScenarioBuilderImpl.java

```diff
@@ -68,11 +68,9 @@ public WorkbenchKieServerPersistentScenarioBuilderImpl() {
         kieApp.getSpec().setCommonConfig(commonConfig);
 
         Server server = new Server();
-        server.addEnvs(authenticationEnvVars);
         kieApp.getSpec().getObjects().addServer(server);
 
         Console console = new Console();
-        console.addEnvs(authenticationEnvVars);
```

Comment:
I guess these changes are intentional. If so, the authenticationEnvVars list is no longer used and it can be removed.